samples/step-by-step/sample.md

```diff
@@ -31,9 +30,25 @@ Completing these steps will serve as an introduction to many important NServiceB
 
 ## Prerequisites
 
-* [.NET Framework Version 4.5.2](https://www.microsoft.com/en-au/download/details.aspx?id=42642)
-* Microsoft Message Queueing (MSMQ) must be [properly installed and configured](/nservicebus/msmq/). The easiest way to do this is using the [Platform Installer](/platform/installer/) tool.
+### .NET Framework Version 4.5.2
+
+[Download the .NET Framework Version 4.5.2](https://www.microsoft.com/en-au/download/details.aspx?id=42642) and install it.
+
+### MSMQ
+
+* Microsoft Message Queueing (MSMQ) must be properly installed and configured. The easiest way to do this is with the appropriate command below from the command line. Alternatively, it can be installed via the Windows GUI following the [MSMQ guide](/platform/installer/).
```

Comment:
How about restructuring this as:

`Installing MSMQ using the command line` and `Installing MSMQ using the Platform Installer.` as two separate headings? Because after I read the `Alternatively ... " then I am immediately shown the command line option which is confusing.